docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using custom redact for Session Replay
+sidebar_order: 5501
+notSupported:
+description: "Learn how to choose which part of your app's data to redact in Session Replay."
+---
+
+<Note>
+
+Using custom redaction for Session Replay can expose sensitive data. Make sure to double-check every part of your app's data that you choose to redact or not.
+
+</Note>
+
+By default, our Session Replay SDK masks all text content, images, and user input, giving you heightened confidence that no sensitive data will leave the device. However, you can choose which parts of your app's data to redact or not by using some different options.
+
+
+## Redact by View class
+
+You can choose which type of view you want to redact or ignore by using the `redactViewClasses` or `ignoreViewClasses` options.
+
+Let's say you have a custom view that you want to redact and a UILabel subclass (which normally would be redacted) that you want to ignore. You can set the options like this:
+
+```swift
+  options.experimental.sessionReplay.redactViewClasses = [MyCustomView.self]
+  options.experimental.sessionReplay.ignoreViewClasses = [MyCustomLabel.self]
+```
```

Comment:
`m`: What drove you to skip adding code samples for ObjC?

Reply:
Im just following the existing Session replay docs that already dont include objc sample